services/api-server/src/simcore_service_api_server/models/schemas/solvers.py

```diff
@@ -65,6 +66,20 @@ def create_from_image(cls, image_meta: ServiceMetaDataPublished) -> "Solver":
             **data,
         )
 
+    @classmethod
+    def create_from_service(cls, service: ServiceGetV2) -> "Solver":
```

Comment:
here is what I mentioned before. The service layer is not suppose to know about schemas
Here the input is a schema from  catalog's client and is converted directly to a schema of the api-server

WE can interate later on this.